packages/common/src/RoundingUtils.ts

```diff
@@ -0,0 +1,31 @@
+import Web3 from "web3";
+import { BigNumber } from "bignumber.js";
+import { BN } from "./types";
+
+const { toBN } = Web3.utils;
+
+/**
+ * @title Utility function for rounding BN values.
+ * @param {BN} inputAmount that is scaled up.
+ * @param {Number} inputDecimals specifies decimal scaling for processed value.
+ * @param {Number} roundingPrecision specifies amount of decimals left after rounding.
+ *      Negative value would round to the nearest 10 to the power of absolute roundingPrecision value.
+ * @param {RoundingMode} roundingMode is optional rounding mode documented in
+ *      https://mikemcl.github.io/bignumber.js/#constructor-properties
+ */
+export function roundToDecimal(
```

Comment:
How are we planning to get this into the price feeds? Will we need to manually add it to each?

Reply:
The plan is for each price feed to add `getCurrentRoundedPrice` and `getHistoricalRoundedPrice` methods that would use results from `getCurrentPrice` and `getHistoricalPrice` respectively and round them using `roundToDecimal` with rounding parameters passed from pf config. This would also require adding rounding parameters for default config based on UMIP specs.

The reason why the rounding cannot be implemented directly in `getCurrentPrice` and `getHistoricalPrice` methods is that expression price feeds need to process unrounded results from other price feeds not to loose precision.

Once `getCurrentRoundedPrice` and `getHistoricalRoundedPrice` methods are implemented in price feeds the final step would be to use them in proposer/dispute bots